apps/src/code-studio/components/progress/MiniViewTopRow.jsx

```diff
@@ -47,11 +47,18 @@ export default class MiniViewTopRow extends React.Component {
 
   render() {
     const {scriptName, linesOfCodeText, selectedSectionId} = this.props;
-
     const sectionId = queryParams('section_id');
-    const params = selectedSectionId
-      ? stringifyQueryParams({section_id: selectedSectionId})
-      : stringifyQueryParams({section_id: sectionId});
+    var params;
+    switch (true) {
+      case typeof selectedSectionId !== 'undefined' && selectedSectionId !== '':
+        params = stringifyQueryParams({section_id: selectedSectionId});
+        break;
+      case typeof sectionId !== 'undefined' && sectionId !== '':
+        params = stringifyQueryParams({section_id: sectionId});
+        break;
+      default:
+        params = '';
```

Comment:
It looks like we are losing the `userId` param when both section and user are selected. could you please take this opportunity to preserve that param? I would suggest finding an example where we preserve all existing params, rather than hand-picking the ones we want to keep.

Reply:
![preserve-user-param](https://user-images.githubusercontent.com/12300669/70671756-f6ee2980-1c31-11ea-96ec-e63f966fd44a.gif)
now I'm only updating the section_id param so user_id (and any other params) is preserved